UN-OCHA/unocha-site
# UNO-808 X icon

feat: replace twitter with x icon in cd footer and social share, regen sprite

Refs: UNO-808


Expected result:

![Selection_085](https://github.com/UN-OCHA/unocha-site/assets/1835923/d021c3a2-b8b1-44d0-b927-146a011a2b9c)

![Selection_087](https://github.com/UN-OCHA/unocha-site/assets/1835923/7764e171-501b-45d9-b2e4-b709a796e4fd)

### To test:
The base theme should already be at v9.2.2 from a previous PR.
1. Checkout this branch, clear cache
2. Check the footer and any page with social share icons.
